tests/testthat/test-Data-validity.R

```diff
@@ -233,7 +233,7 @@ test_that("validate_data_DA returns error for Tmax of wrong length and negative
   )
 })
 
-test_that("validate_data_DA returns error for vector u of wrong length and values", { # nolintr
+test_that("validate_data_DA: error for u of wrong length and values", {
```

Comment:
ok for here. generally I prefer when there is a meaningful sentence like statement in the tests description instead of a strict schema like "fun: error..." because the former is more readable.